package.json

```diff
@@ -30,26 +30,28 @@
     "url": "https://github.com/canjs/bit-docs-html-canjs/issues"
   },
   "homepage": "https://github.com/canjs/bit-docs-html-canjs#readme",
-  "system": {},
```

Comment:
Was this configuration not required? https://stealjs.com/docs/StealJS.guides.substitution_conditional_loading.html#set-up-steal-conditional